### ISSUE TITLE
ssd-shufflenet mAP ?

### ISSUE THREAD

**Issue by ananddb90** [linchaozhang/shufflenet-ssd#2]
Thank you for your work. Do you have training results of SSD detection on shuffle-net because I would like to know mAP
thank you

**Comment by linchaozhang** [comment]
@ananddb90 not good, only 63% on VOC, still in finetune. how about you?

**Comment by ananddb90** [comment]
I tried with RFCN and faster-RCNN and I got very less on VOC (~43%) and story was when I used mobilenet. where as ssd on mobile net performed really good (~72) . have faced similar issues before ?
Also could you please tell me how you are planning to fine tune further ? may be I am missing something 
regards,
anand

**Comment by zoufangyu1987** [comment]
@linchaozhang  @ananddb90 
Do you have 0.5_ShuffleNet or 0.75_ShuffleNet pre_trained caffe model ? and its train.prototxt?
Thank you very much!

**Comment by ananddb90** [comment]
@zoufangyu1987  sorry I don't have 0.5_ShuffleNet or 0.75_ShuffleNet models

**Comment by felixfuu** [comment]
@linchaozhang @ananddb90  I also trained the ssd_shuflenet on voc2007,and my mAP is 53%. But it runs slower than vgg16 when image size is 300*300, why?

**Comment by CrazySssst** [comment]
@linchaozhang  Do you pre-train the model in coco?

**Comment by linchaozhang** [comment]
@CrazySssst i have pre-trained on coco, 63% is the result after pre-train on coco.

**Comment by ghost** [comment]
Hi, is there any chance you can provide the pretrained models?

**Comment by share020** [comment]
@ananddb90 @linchaozhang Hello, have you succeed in training the shufflenet-ssd with this code? I tried   it with pascal voc dataset and the model didn't converge after 120000 iteration. Could you please tell me what's wrong and provide your pretrained model? Thanks.

**Comment by zfc929** [comment]
I train my data ,the map is 0.658, but the small object is not well ,who train model with 512 *512 ?

**Comment by gjm441** [comment]
I trained shufflenet-ssd on widerface, easy and medium AP only got 0.6 .It got bad performance, the trainning loss stayed at 3.7 at last and hard to descend .May be shufflenet not suitable for detection? @linchaozhang  give some suggestions?
